pkg/printer/workload_status_printer.go

```diff
@@ -115,15 +115,18 @@ func WorkloadIssuesPrinter(w io.Writer, workload *cartov1alpha1.Workload) error
 		return nil
 	}
 	printIssues := func(workload *cartov1alpha1.Workload, _ table.PrintOptions) ([]metav1beta1.TableRow, error) {
-		readyRow := metav1beta1.TableRow{
-			Cells: []interface{}{
-				fmt.Sprintf("Workload [%s]:", readyCondition.Reason),
-				readyCondition.Message,
-			},
+		rows := []metav1beta1.TableRow{}
+		if strings.TrimSpace(readyCondition.Message) != "" {
+			readyRow := metav1beta1.TableRow{
+				Cells: []interface{}{
+					fmt.Sprintf("Workload [%s]:", readyCondition.Reason),
```

Comment:
create const or use one if there is for kind Workload in pkg cartov1alpha1

Reply:
Addressed in PR #261